webroot/src/pages/SelectPrivileges/SelectPrivileges.less

```diff
@@ -0,0 +1,198 @@
+//
+.select-privileges-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2rem;
+
+    .jurisprudence-modal {
+        :deep(.modal-container) {
+            max-width: 60rem;
+            padding: 2rem;
+
+            @media @tabletWidth {
+                padding: 4rem;
+            }
+
+            .jurisprudence-modal-content {
+                padding-top: 1rem;
+                overflow-y: hidden;
+
+                .action-button-row {
+                    display: flex;
+                    flex-direction: row;
+                    justify-content: flex-end;
+                    margin-top: 4rem;
+                    margin-bottom: -3rem;
+
+                    .understand-button {
+                        margin-left: 1rem;
+                    }
+                }
+            }
+        }
+    }
+
+    .privilege-form {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        max-width: 72rem;
+
+        @media @desktopWidth {
+            max-width: unset;
+        }
+
+        .button-row {
+            display: flex;
+            flex-direction: row;
+            align-items: flex-end;
+            justify-content: space-between;
+            width: 100%;
+
+            .right-cell {
+                display: flex;
+                flex-direction: row;
+                align-items: flex-end;
+                justify-content: space-between;
+
+                .back-button {
+                    margin-right: 1rem;
+                }
+            }
+        }
+
+        .select-privileges-core-container {
+            margin-top: -1rem;
+            padding: 2rem;
+            border-radius: 20px;
+            background-color: white;
+
+            @media @tabletWidth {
+                width: 72rem;
+            }
+
+            @media @desktopWidth {
+                width: unset;
+                min-width: 72rem;
+                margin-top: 8rem;
+            }
+
+            .select-privileges-title {
+                margin: 0 1rem 2rem 1rem;
+                font-size: @fontSizeLarger;
+            }
+
+            .lists-container {
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+
+                @media @tabletWidth {
+                    flex-direction: row;
+                    align-items: flex-start;
+                    justify-content: space-between;
+                }
+
+                .state-select-list {
+                    display: flex;
+                    flex-flow: column wrap;
+
+                    @media @desktopWidth {
+                        max-height: 65vh;
+                    }
+
+                    .state-select-unit {
+                        width: 27rem;
+                        margin: 1rem;
+                        padding: 0 1rem;
+                        border: 1px solid @lightGrey;
+                        border-radius: 10px;
```

Comment:
Now that this element is more expectedly interactive, a `cursor: pointer` might be in order.

Reply:
Resolved.